src/clusterfuzz/_internal/bot/fuzzers/googlefuzztest/engine.py

```diff
@@ -26,6 +26,16 @@ class GoogleFuzzTestError(Exception):
   """Base exception class."""
 
 
+class GoogleFuzzTestFuzzOptions(engine.FuzzOptions):
```

Comment:
Why are we making this change?

Reply:
This was the cleanest way I could come up with to pass environment variables to the binary under test, so I could force the execution to crash or not. This does not make sense anymore, given that we gave up on having test binaries for googlefuzztest.